test/fixtures/omniauth_response.json

```diff
@@ -0,0 +1,32 @@
+{
+    "provider": "github",
+    "uid": "12345678",
+    "info": {
+      "nickname": "exampleuser",
+      "email": "user@example.com",
+      "image": "https://avatars.githubusercontent.com/u/12345678?v=4"
+    },
+    "credentials": {
+      "token": "//encrypted//"
+    },
+    "extra": {
+      "raw_info": {
+        "login": "exampleuser",
+        "avatar_url": "https://avatars.githubusercontent.com/u/12345678?v=4",
+        "url": "https://api.github.com/users/exampleuser",
+        "html_url": "https://github.com/exampleuser"
+      },
+      "all_emails": [
+        {
+          "email": "12345678+exampleuser@users.noreply.github.com"
+        },
+        {
+          "email": "example@user.com"
```

Comment:
"Fake" email addresses for testing can occasionally cause real world issues if you're not careful.

This actually happened at my job within the last couple of months. Our QA team was creating fake accounts to manually test user flows in the pre-prod environment. Our automated email system ended up spamming some random person who eventually contacted us to stop using their domain.

I only mention this because [user.com](https://www.user.com/) is a real site. Since we aren't sending email here it's not a big deal, but it's worth mentioning for future testing.

I like to use [example.com](https://example.com/) because it explicitly states:

>This domain is for use in illustrative examples in documents. You may use this domain in literature without prior coordination or asking for permission.

Reply:
Noted, that makes sense. Good to know for writing future testing data!